sqlglot/dialects/presto.py

```diff
@@ -354,6 +354,7 @@ class Generator(generator.Generator):
             exp.TimeStrToUnix: lambda self, e: f"TO_UNIXTIME(DATE_PARSE({self.sql(e, 'this')}, {Presto.TIME_FORMAT}))",
             exp.TimeToStr: lambda self, e: f"DATE_FORMAT({self.sql(e, 'this')}, {self.format_time(e)})",
             exp.TimeToUnix: rename_func("TO_UNIXTIME"),
+            exp.ToDays: transforms.preprocess([transforms.todays_to_datediff]),
```

Comment:
in general, let's avoid transforms except for big heavy machinary like SELECT transforms.

for this one, let's just add a dialect generator function

Reply:
roger